string/readme.md

```diff
@@ -11,5 +11,11 @@ A string is a class that contains a char array, but automatically manages it for
 
 <hr>
 
-## Questions :
+# Minimum number of deletions to make pallindrome:
+
+There are two methods:
```

Comment:
Minimum number of deletions to make a string palindrome --------> <your code hyperlink with language name>

Reply:
Then what can be written?? Since it is not a script where we can write how to run it. So what can be written? Can you suggest me.